src/invision/utils/movie-converter.util.js

```diff
@@ -0,0 +1,12 @@
+export const movieConverter= (movie)=>{
```

Comment:
I like your **movieMapper** and **movieConverter**

Reply:
thanks